kDrive/UI/Controller/MainTabViewController.swift

```diff
@@ -45,9 +45,21 @@ class MainTabViewController: UITabBarController, Restorable, PlusButtonObserver
         super.init(nibName: nil, bundle: nil)
         viewControllers = rootViewControllers
 
+        // Use selectedIndex argument if any
         if let selectedIndex {
             self.selectedIndex = selectedIndex
         }
+
+        // TODO: Abstract
```

Comment:
TODO: This should be abstracted away from the init method, the correct index should be passed at init form reading the scene.